chart/NOTES.txt

```diff
@@ -0,0 +1,9 @@
+Thank you for installing {{ .Chart.Name }}.
+
+Your release is named {{ .Release.Name }}.
+
+To learn more about the release, try:
+
+  $ helm status {{ .Release.Name }}
+  $ helm get all {{ .Release.Name }}
```

Comment:
I think some useful links to the docs and to where to post issues would be good :)